### ISSUE TITLE
Login failed 

### ISSUE THREAD

**Issue by Enerccio** [CastagnaIT/plugin.video.netflix#834]
#### Your Environment
- Netflix add-on version: 1.0.1
- Operating system version/name: Android 8
- Device model:  TCL Percee (tv)

Used Operating system:
* [X ] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

### Describe the bug

Can't login with correct username and password, not blocked on netflix and accessing from same device and same ip as normal tv netflix app that works. Password and email are correct. 

#### Expected behavior

I log in.

#### Actual behavior

I get login error

#### Steps to reproduce the behavior

1. Install from your repository
2. Launch netflix addon
3. Enter login credentials (via adb shell input text)

### Debug log

The debug log can be found from this link: https://paste.kodi.tv/yozoxaqehu

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source

**Comment by Enerccio** [comment]
duplicate of what? And are you planning to fix it?
